config/typescript/tsconfig.base.json

```diff
@@ -1,33 +1,15 @@
 {
-  "compileOnSave": false,
+  "extends": "@shopify/typescript-configs/library.json",
   "compilerOptions": {
     "composite": true,
-    "declaration": true,
-    "declarationMap": true,
-    "pretty": true,
-    "moduleResolution": "node",
-    "target": "es5",
-    "downlevelIteration": true,
-    "esModuleInterop": true,
-    "jsx": "react",
-    "isolatedModules": true,
-    "strictNullChecks": true,
+    // Strict mode enables these strict* and noImplicit* options.
```

Comment:
While this obviously seems to work fine, could we perhaps use a hack like a `"_NOTICE": "...."` to keep this as valid JSON?

Reply:
Updated!

Reply:
Ah nope. TS type check says no:

```
config/typescript/tsconfig.base.json:6:5 - error TS5023: Unknown compiler option '__TODOCOMMENT2'.

6     "__TODOCOMMENT2": "Disable them temporarily while we fix up problematic behaviour from the time before we wanted strict mode",
      ~~~~~~~~~~~~~~~~
```